core/domain/state_domain.py

```diff
@@ -797,7 +797,13 @@ class WrittenTranslations(python_utils.OBJECT):
     """
 
     def __init__(self, translations_mapping):
-        """Initializes a WrittenTranslations domain object."""
+        """Initializes a WrittenTranslations domain object.
+
+        Args:
+            translations_mapping: dict. Contains content_id and a list
```

Comment:
"The keys of this dict are content_ids, and the corresponding values are a list of ..."

Also, I'm not really sure what "a list of language codes to be translated" means. Does it represent languages for which the content is already translated, or languages which the content still needs to be translated into, or all languages that Oppia supports, or something else? Adding @DubeySandeep for clarity/confirmation -- please could you confirm that this docstring is correct / suggest ways to make it clearer?

Reply:
```
translations_mapping_dict = {
  'content_id_1': {
    'language_code_1': WRITTEN_TRANSLATION_DICT,
    'language_code_2': WRITTEN_TRANSLATION_DICT
  },
  'content_id_2': {
    'language_code_1': WRITTEN_TRANSLATION_DICT,
    'language_code_2': WRITTEN_TRANSLATION_DICT
  }
```

This mapping represents:
There exist a translation in `language_code_1` and `language_code_2` for the  content with content_id `content_id_1`.
There exist a translation in `language_code_1` and `language_code_2` for the  content with content_id `content_id_2`.

Note: WRITTEN_TRANSLATION_DICT is the dict representation of WrittenTranslation object which consists of `html` and `needs_update` attributes.

Reply:
Sounds good, thanks @DubeySandeep. @edeediong could you please update the docstring accordingly?

Reply:
Re-flagging this comment -- based on Sandeep's response, language codes is not a list.